app/flatpak-builtins-build-update-repo.c

```diff
@@ -655,14 +655,18 @@ flatpak_builtin_build_update_repo (int argc, char **argv,
                               cancellable, error))
         return FALSE;
 
-      formatted_freed_size = g_format_size_full (objsize_total, 0);
 
-      g_print (_("Total objects: %u\n"), n_objects_total);
-      if (n_objects_pruned == 0)
-        g_print (_("No unreachable objects\n"));
-      else
-        g_print (_("Deleted %u objects, %s freed\n"),
-                 n_objects_pruned, formatted_freed_size);
+      if (!opt_prune_dry_run)
+        {
+          formatted_freed_size = g_format_size_full (objsize_total, 0);
```

Comment:
The declaration of the `formatted_freed_size` variable should be moved to this if block now.

But, what's the reasoning for not printing this on dry runs? Does it always return zero or a bogus value on dry runs?

The commit message could have some of that reasoning.